Drivers/Src/MotorController.c

```diff
@@ -27,6 +94,8 @@ void MotorController_Init(){
  * @return  None
  */ 
 void MotorController_Drive(float newVelocity, float motorCurrent){
+    CPU_TS ts;
+	OS_ERR err;
 
     uint32_t nv = *((uint32_t *)((void *) &newVelocity));
     uint32_t mc = *((uint32_t *)((void *) &motorCurrent));
```

Comment:
I feel like the code for copying the floats into an array could be better. Maybe something like
```
uint8_t data[8];
memcpy(data, &motorCurrent, sizeof(motorCurrent));
memcpy(data + sizeof(motorCurrent), &newVeloctiy, sizeof(newVelocity));
```